library/src/main/java/org/apache/apex/malhar/lib/dedup/TimeBasedDedupOperator.java

```diff
@@ -186,7 +186,7 @@ public void deactivate()
   @Override
   protected void putManagedState(Object tuple)
   {
-    ((ManagedTimeUnifiedStateImpl)managedState).put(getTime(tuple), getKey(tuple), new Slice(new byte[0]));
+    ((ManagedTimeUnifiedStateImpl)managedState).put(getTime(tuple), getKey(tuple), new Slice(null));
```

Comment:
We cannot create a separate PR as Apex 3.4.0 was using netlet 1.2.1 which will throw NullPointerException if null is passed.